sdk/python/kfp/compiler_cli_tests/test_data/lightweight_python_functions_v2_pipeline.py

```diff
@@ -140,4 +140,4 @@ def pipeline(message: str, input_dict: Dict[str, int] = {'A': 1, 'B': 2}):
 
 if __name__ == '__main__':
     compiler.Compiler().compile(
-        pipeline_func=pipeline, package_path=__file__.replace('.py', '.json'))
```

Comment:
Let's also delete all the old json golden files.

Reply:
Done